ember-data/2017-11/november-01.md

```diff
@@ -0,0 +1,8 @@
+# ember-data Meeting Notes - 2017-11-01
+
+## Attendees
+
+## Agenda
```

Comment:
Reply here with agenda items

Reply:
Follow up on solving various use cases unload via addon

Reply:
in particular
 - addon for transitive-unload
 - addon for soft-delete

Reply:
Discuss testing API's:

* https://github.com/emberjs/data/issues/5247 - `store.createRecord` should be async?
* Meta issue around creating new-style `setupSerializerTest(hooks)` (as pseudo coded in [my blog post](http://rwjblue.com/2017/10/23/ember-qunit-simplication/#why))

Reply:
Discuss https://github.com/emberjs/data/pull/5230 and whether/how to support multi-table inheritance.

Reply:
Potentially discuss allowing unique indexes against simple (non-compound) attributes